### PR TITLE
Feature/administration page

### PR DESCRIPTION
**Pull request by JeanVittory**
## Summary

- Router links were built using react router dom, admin dashboard was included.

## To-Do
- The next step is starting the crud of products and doctors

### CODE REVIEW

File: src/components/navigationLink/NavigationLink.components.jsx [hunk 1]
```diff
@@ -1,17 +1,24 @@
...
 import { Link } from 'react-router-dom';
+import { PublicRoutes, PrivateRoutes } from '@routes';
 import './navigationLink.components.scss';
 
 export const NavigationLink = ({ text, isButton }) => {
   const regex = /\s/i;
   const link = text.replace(regex, '-').toLowerCase();
+  const route =
+    text === 'Your Panel'
+      ? '/home/admin/dashboard'
```
Comment (ht1204) [inline src/components/navigationLink/NavigationLink.components.jsx:11]: This route could be better in a constant variable, please make that change, about the rest, great job.

**Review by ht1204 — APPROVED** [review]
Please, consider the comment that I left, approve with changes to be done before merge.

**Review by ht1204 — APPROVED** [review]

**Review by ht1204 — CHANGES_REQUESTED** [review]
This is Still WIP, this could be a draft PR yet, attentive to updates

File: src/components/navigationLink/NavigationLink.components.jsx (hunk 1 above)
Comment (JeanVittory) [reply src/components/navigationLink/NavigationLink.components.jsx]: This recommendation was resolved in commit number c762457

**Review by ht1204 — APPROVED** [review]

**Pull request merged**